### PR TITLE
feat(monorepo): added readme for packages

### PR DESCRIPTION
**Pull request by prodesert22**
## Summary

- Added readme for packages

### Packages

- [x] Airdrop
- [x] Bridge
- [x] eslint-config-custom
- [x] Governance
- [x] honeycomb-provider
- [x] Pools
- [x] Portfolio
- [x] rollup-plugin-tsc-alias
- [x] Sar
- [x] Shared
- [x] State-Hooks
- [x] Swap
- [x] Token-Drawer
- [x] Wallet-connectors
- [x] Wallet modal

### CODE REVIEW

File: monorepo/packages/airdrop/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      </Web3ProviderNetwork>
+    </Web3ReactProvider>
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
+```
+
+## Story Book
+
+1. do `yarn storybook` to start the storybook
```
Comment (HariSeldon23) [inline monorepo/packages/airdrop/README.md:65]: Does this mean that for each package we have to run Storybook? That means we won't be able to run a single Storybook and have all packages run in Storybook at the same time?

Currently all Components are deployed at components.pangolin.exchange. Is this no longer possible?

File: monorepo/packages/core/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+or
+
+`npm install @pangolindex/core`
+
+### Install below dependancies as its peer dependancies
+
+```
+react
+react-dom
+@pangolindex/sdk
```
Comment (HariSeldon23) [inline monorepo/packages/core/README.md:16]: @SarjuHansaliya @prodesert22 does it make sense to also move this to Honeycomb Github org?

For example, how would we store Ortege chain details? We can't store them in the Pangolin SDK

**Review by HariSeldon23 — APPROVED** [review]

File: monorepo/packages/airdrop/README.md (hunk 1 above)
Comment (prodesert22) [reply monorepo/packages/airdrop/README.md:65]: Now yes, I think in future is possible, @SarjuHansaliya can give your more details

File: monorepo/packages/core/README.md (hunk 2 above)
Comment (prodesert22) [reply monorepo/packages/core/README.md:16]: Yes we will move to honeycomb, are a task about this
https://github.com/pangolindex/components/pull/404

Comment (SarjuHansaliya) [reply monorepo/packages/core/README.md:16]: @HariSeldon23 ortege sdk will be under `@ortege/sdk` and chain details we will store in ortege sdk.

File: monorepo/packages/airdrop/README.md (hunk 1 above)
Comment (SarjuHansaliya) [reply monorepo/packages/airdrop/README.md:65]: yes storybook will be separate for now but I think there is composition feature in storybook where we can merge all stories but that we will handle once all essential things are done.

**Pull request merged**